provider/README.md

```diff
@@ -336,8 +336,52 @@ When multiple query parameters are specified, they should all apply to the retur
 [Top][toc]
 
 ## Realtime Data
+All providers should provide an endpoint that displays the current status of a providers vehicles on the PROW. Only vehicles that are currently in an available, unavailable, or reserved states should be returned in this payload. Data in this endpoint should generally reconcile with data from the status changes enpdoint.
```

Comment:
I feel like there maybe needs to be more definition about how this relates to the status changes endpoint. Specifically, I wonder about how this endpoint deals with lost vehicles:
- If the provider is allowed to omit a vehicle from this endpoint when they no longer believe it's in a knowable location on the PROW, it won't be possible in general to reconcile this data with data from the status changes endpoint, since that endpoint has no way of representing a vehicle going missing.
- If it's mandatory for the data from this endpoint to correlate with the status changes endpoint, the provider will have no way to prune missing vehicles, resulting in a monotonically increasing number of "stuck" vehicles in this endpoint over time.

The former option definitely seems more practical, but it also creates a weird situation where the only way for a consumer to know when a provider considers a vehicle "lost" is to watch the realtime endpoint and see when it disappears. If removing vehicles from the realtime endpoint is permissible, it seems like it might make sense to also add a status change capturing that removal so that the two streams can be reliably correlated with each other.

Reply:
This is the main issue to be resolved here. There have been long running conversations about how to deal with vehicles that go missing, or leave the PROW without an event. 

I've always advocated for the addition of a `lost` event type reason for `removed` which would make the latter option you suggest feasible. We've gone back and forth on the justification for such a reason, and whether or not this would obfuscate valuable data from consumers.

Reply:
What does it mean for a reserved state to be returned in this payload? Does that mean the endpoint is returning the location where a reserved event occurred or is that the realtime location of the vehicle while it is in a trip? If the latter, I'm wondering if there needs to be some language about time resolution for two reasons: this is a moving vehicle and there may be some privacy considerations. That is, this would be realtime data of a person in a trip as opposed to a static vehicle on the street not associated with an individual.

Reply:
I think `removed`/`lost` should be either `inactive`/`lost` or `missing`/`lost`.  My interpretation of `removed` is "not in the public right of way because the provider has it".  `lost` is pretty clearly "the provider doesn't have it", and it _may or may not_ be in the public right of way.

Reply:
On the 2/13 call we decided to punt `removed`/`lost` to the Agency/Provider alignment work in #271 slated for the next major MDS release.